packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -409,18 +409,14 @@ const TooltipContentHoverable = React.forwardRef<
 
   const handleCreateGraceArea = React.useCallback(
     (event: PointerEvent, hoverTarget: HTMLElement) => {
-      const currentTarget = event.currentTarget as HTMLElement;
-      const exitPoint = { x: event.clientX, y: event.clientY };
-      const exitSide = getExitSideFromRect(exitPoint, currentTarget.getBoundingClientRect());
-
-      const bleed = exitSide === 'right' || exitSide === 'bottom' ? -5 : 5;
-      const isXAxis = exitSide === 'right' || exitSide === 'left';
-      const startPoint = isXAxis
-        ? { x: event.clientX + bleed, y: event.clientY }
-        : { x: event.clientX, y: event.clientY + bleed };
-
       const hoverTargetPoints = getPointsFromRect(hoverTarget.getBoundingClientRect());
-      const graceArea = getHull([startPoint, ...hoverTargetPoints]);
+      const graceArea = getHull([
+        { x: event.clientX + 5, y: event.clientY + 5 },
+        { x: event.clientX + 5, y: event.clientY - 5 },
+        { x: event.clientX - 5, y: event.clientY + 5 },
+        { x: event.clientX - 5, y: event.clientY - 5 },
```

Comment:
This is smart as the extra points will be swallowed by the hull so it can be lazy and not side-dependent. 👍

Reply:
The only caveat I see where we might still want to bring back some side-dependent checks is this:

<img width="374" alt="CleanShot 2023-02-16 at 12 24 18@2x" src="https://user-images.githubusercontent.com/1539897/219363972-3ae33023-8c1e-4194-a754-c258c73b0f37.png">

as admittedly, exiting from this side should be able to dismiss instantly and not have to suffer the extra 5px wait as we know it's the wrong direction anyway.

Reply:
Does that make sense @lokeswaran-aj?

Reply:
Hi, @benoitgrelard
yes, I get it. Right we have an extra 5px on the side and down also. Do you want to instantly exit only on the sides or down also?

Reply:
Yeah essentially we should only add extra points relative to the exit side.

I think basically it's almost the same as what we currently have, but instead of just one point which we pull back towards the opposite side, we'd add 2 points.

To give a concrete example:

- if we exist on the "top" side
- we'd add 2 points:
  - `{ x: event.clientX - 5, y: event.clientY - 5 }`
  - `{ x: event.clientX + 5, y: event.clientY - 5 }`

Does that make sense?

Reply:
Essentially given a side:
- the 2 points should extend along that side in both directions
- the 2 points should be pushed towards the the opposite side